recipes/libtiff/all/conanfile.py

```diff
@@ -121,14 +127,16 @@ def generate(self):
             deps.set_property("xz_utils", "cmake_target_name", "liblzma::liblzma")
             deps.set_property("libdeflate", "cmake_file_name", "Deflate")
             deps.set_property("libdeflate", "cmake_target_name", "Deflate::Deflate")
+        deps.set_property("lerc", "cmake_target_name", "LERC::LERC")
```

Comment:
Does it hard fail if with_lerc=False?

Reply:
No, it's not strict about the target names.